internal/logger/logger.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 )
 
-var debugging = false
+const debugging = false
```

Comment:
if its a const then a user cant modify this via flags.

Reply:
The user already can't modify it via flags? Its a private variable.

As built right now, only a developer can change this by editing the variable.